docs/manual_testing/performance_testing/load_testing.rst

```diff
@@ -0,0 +1,47 @@
+
+Kolibri Load‑Testing CLI — Quick Start
+======================================
+
+Run from ``integration_testing/load_testing/``; entrypoint is ``loadtest.py``.
+
+Help
+----
+.. code-block:: bash
+
+   python loadtest.py --help
+   python loadtest.py --help
+   python loadtest.py run --help
+
+See the help for available flags (e.g., users, spawn rate, duration, headless, retries).
+
+Full provision → manual capture → test
+--------------------------------------
+Runs the whole flow: provision device → create facility/users/class → import QA channel → create lesson → open
+a browser for **manual capture** → save a versioned HAR → execute the Locust test.
+
+.. code-block:: bash
+
+   python loadtest.py
+
+Full run using an existing HAR (skip capture)
+---------------------------------------------
+Use a pre-recorded HAR to skip the manual capture step. With a file in ``har_files/lesson_flow_kolibri_testing_high_latency.har``, for example:
+
+.. code-block:: bash
+
+   python loadtest.py --har har_files/lesson_flow_kolibri_testing_high_latency.har
+
+Other flags
+-----------
+
+Non-interactive server and credentials:
+
+.. code-block:: bash
+
+   python loadtest.py --server http://127.0.0.1:8080 --username admin --password sosecure
```

Comment:
I think this should be `http://127.0.0.1:8000` ?

Reply:
One would usually want to load test on the production server rather than the development server, hence the 8080 - but seems like that should be explained :)

Reply:
ah, yes that makes sense. adding a note there to explain would help i think